.github/workflows/saucelabs-UI-tests.yml

```diff
@@ -38,7 +38,7 @@ jobs:
           MATCH_USERNAME: ${{ secrets.MATCH_USERNAME }}
       
       - name: Archiving DerivedData
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@v3
```

Comment:
`m`: Ideally, we could do these updates in another PR so we can merge it sooner.

Reply:
Here you go: https://github.com/getsentry/sentry-cocoa/pull/1879